client/src/components/ConvoCard/ConvoCard.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import Box from '@material-ui/core/Box';
+import Badge from '@material-ui/core/Badge';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import ConvoContent from '../ConvoContent/ConvoContent';
+import { useAuth } from '../../context/useAuthContext';
+
+const ConvoCard = (): JSX.Element => {
+  const [online, setOnline] = useState<boolean>(false)
+	const classes = useStyles();
+    const { loggedInUser } = useAuth();
+
+  const selectConversation = () => {
+    //set conversation state
+  }
+
+	return (
+		<Box className={classes.root} onClick={selectConversation}>
```

Comment:
Please fix spacing/indentation issues. You can configure prettier (code formatter) to format on save :)